gemsfx/src/main/java/com/dlsc/gemsfx/skins/StripViewSkin.java

```diff
@@ -87,13 +89,60 @@ public StripViewSkin(StripView<T> strip) {
 
         strip.itemsProperty().addListener((Observable it) -> buildContent());
         buildContent();
+
+        strip.addEventFilter(KeyEvent.KEY_PRESSED, this::handleKeyPress);
     }
 
+    private void handleKeyPress(KeyEvent event) {
+        List<T> itemsList = getSkinnable().getItems();
+        T currentSelectedItem = getSkinnable().getSelectedItem();
+        int index = itemsList.indexOf(currentSelectedItem);
+        int itemCount = itemsList.size();
+
+        switch (event.getCode()) {
+            case RIGHT:
+            case ENTER:
+                // Check if loop selection is enabled or if we haven't reached the last item yet
+                if (getSkinnable().isLoopSelection() || index < itemCount - 1) {
```

Comment:
Replace all `getSkinnable()` calls in this method with a previously assigned variable please.